src/mainviews/job-detail.tsx

```diff
@@ -120,201 +96,235 @@ export function JobDetail(props: IJobDetailProps): JSX.Element {
-            />
-            <TextFieldStyled
-              label={trans.__('Status')}
-              defaultValue={job?.status ?? ''}
-            />
-            <Accordion defaultExpanded={true}>
-              <AccordionSummary
-                expandIcon={<caretDownIcon.react />}
-                aria-controls="panel-content"
-                id="panel-header"
-              >
-                <FormLabel component="legend">
-                  {trans.__('Parameters')}
-                </FormLabel>
-              </AccordionSummary>
-              <AccordionDetails id="panel-content">
-                <Stack spacing={4}>
-                  {Object.entries(job?.parameters ?? {}).map(
-                    ([parameter, value]) => (
-                      <Stack key={parameter} direction="row" spacing={1}>
-                        <TextFieldStyled
-                          label={trans.__('Parameter')}
-                          defaultValue={parameter}
-                          InputProps={{
-                            readOnly: true
-                          }}
-                        />
-                        <TextFieldStyled
-                          label={trans.__('Value')}
-                          defaultValue={value}
-                        />
-                      </Stack>
-                    )
-                  )}
-                </Stack>
-              </AccordionDetails>
-            </Accordion>
-          </Stack>
+              <Stack direction={'row'} gap={2} flexWrap={'wrap'}>
+                {outputFormatsStrings.map(outputFormatString => (
+                  <OutputFile
+                    outputType={outputFormatString}
+                    app={props.app}
+                    outputPath={props.model.outputPath}
+                  />
+                ))}
+              </Stack>
+            </>
+          )}
+        </Stack>
+      </CardContent>
+    </Card>
+  );
 
-          <Accordion defaultExpanded={true}>
-            <AccordionSummary
-              expandIcon={<caretDownIcon.react />}
-              aria-controls="panel-content"
-              id="panel-header"
-            >
-              <FormLabel component="legend">
-                {trans.__('Additional options')}
-              </FormLabel>
-            </AccordionSummary>
-            <AccordionDetails id="panel-content">
-              <Stack spacing={4}>
-                <TextFieldStyled
-                  label={trans.__('Create time')}
-                  defaultValue={timestampLocalize(job?.create_time ?? '')}
-                />
+  const Parameters = () => (
+    <Card>
+      <CardContent>
+        <FormLabel sx={{ mb: 4 }} component="legend">
+          {trans.__('Parameters')}
+        </FormLabel>
+        <Stack spacing={4}>
+          {props.model.parameters &&
+            props.model.parameters.map(parameter => (
+              <Stack direction={'row'} gap={2} flexWrap={'wrap'}>
```

Comment:
There should be a unique `key` for each component created this way. You can do this by adding the index (`idx`) to the `map` function:

```suggestion
            props.model.parameters.map((parameter, idx) => (
              <Stack key={idx} direction={'row'} gap={2} flexWrap={'wrap'}>
```

Reply:
Thank you!